content/docs/getting-started/hello-features.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Tokio Features"
+weight : 1020
+menu:
+  docs:
+    parent: getting_started
+---
+
+Feel free to skip this section, and just use `features = ["full"]` while
+learning. This section provides some examples of how and when to specify
+tokio features. 
+
+## Often using all features is fine 
+
+When writing an app, we'll often use all of Tokio's features to accelerate
+our development time:
+
+```toml
+[dependencies]
+tokio = { version = "0.2", features = ["full"] }
+```
```

Comment:
I suggest adding filenames before each code example. E.g.
````md
Cargo.toml:
```toml
  [dependencies]
  tokio = { version = "0.2", features = ["full"] }
```
````

Reply:
I like this idea.